src/tests/Navigation.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+// eslint-disable-next-line
```

Comment:
- [ ] Please don't disable eslint checks. We use Eslint to keep to a certain standard of the Javascript we write and disabling a check deviates from that.